pom.xml

```diff
@@ -33,7 +33,7 @@
     <argLine/>
     <bulk-operations.yaml.file>${project.basedir}/src/main/resources/swagger.api/bulk-operations.yaml</bulk-operations.yaml.file>
 
-    <folio-spring-base.version>5.0.2</folio-spring-base.version>
+    <folio-spring-base.version>6.0.0</folio-spring-base.version>
     <jackson-bom.version>2.14.1</jackson-bom.version>
```

Comment:
did you check other dependenies' updates? Looks like at least spring-kafka.version should have third version released